firstvoices/backend/tests/test_apis/base_media_test.py

```diff
@@ -296,6 +323,77 @@ def assert_related_objects_deleted(self, instance):
         self.assert_instance_deleted(instance.small)
         self.assert_instance_deleted(instance.thumbnail)
 
+    def assert_secondary_fields(self, expected_data, updated_instance):
+        assert updated_instance.description == expected_data["description"]
+        assert updated_instance.acknowledgement == expected_data["acknowledgement"]
+        assert updated_instance.exclude_from_kids == expected_data["excludeFromKids"]
+        assert updated_instance.exclude_from_games == expected_data["excludeFromGames"]
+        assert updated_instance.is_shared == expected_data["isShared"]
+
+    def assert_patch_instance_updated_fields(self, data, updated_instance):
+        assert updated_instance.title == data["title"]
+
+    def assert_response(self, expected_data, actual_response):
+        assert actual_response["title"] == expected_data["title"]
+        assert actual_response["description"] == expected_data["description"]
+        assert actual_response["acknowledgement"] == expected_data["acknowledgement"]
+        assert actual_response["excludeFromKids"] == expected_data["excludeFromKids"]
+        assert actual_response["excludeFromGames"] == expected_data["excludeFromGames"]
+        assert actual_response["isShared"] == expected_data["isShared"]
+
+        expected_file_path = (
+            expected_data["original"].content.url
+            if hasattr(expected_data["original"], "content")
+            else expected_data["original"].name
+        )
+        expected_filename = expected_file_path.split(".")[0]
+        expected_file_extension = expected_file_path.split(".")[1]
+        actual_filename = actual_response["original"]["path"].split(".")[0]
+        actual_file_extension = actual_response["original"]["path"].split(".")[1]
+        assert expected_filename in actual_filename
+        assert expected_file_extension in actual_file_extension
+
+        if self.__class__.__name__ == "TestAudioEndpoint":
```

Comment:
This doesn't belong in the base class. A better way to accomplish this would be to override this method in the audio test class, and have it call the super method and then check the speaker data